spec/support/account_helper.rb

```diff
@@ -0,0 +1,15 @@
+module AccountHelper
+
+  def login_with(email, password = "secret")
```

Comment:
I'd look into the [Warden Test Helpers](https://github.com/heartcombo/devise/wiki/How-To:-Test-with-Capybara) as a way to login without running through these http requests every time